Add proper grid alignment for rows in rule details

vorillaz opened the pull request:
## Description

This one fixes some alignment issues regarding the threshold values. Tags may get quite lengthy breaking the lines between the columns, thus a grid is required in order to create a more structured way for displaying the rule details.

### Screenshots:

#### Before:
![Screenshot 2020-09-14 at 1 43 44 PM](https://user-images.githubusercontent.com/1022166/93079758-49fb7e00-f695-11ea-8e05-9bb068d440de.png)

#### After:
![Screenshot 2020-09-15 at 12 27 04 PM](https://user-images.githubusercontent.com/1022166/93192872-d027cb00-f74e-11ea-9e5a-5a72e30dcc3e.png)

### Checklist:

- [x] Make sure you give an high-level overview of the problem & the solution
- [x] Make sure the related issue is referenced
- [x] Make sure screenshots are added if the UI is changed in any way

kostaspap commented:
Thanks for fixing this! I would recommend getting a code review from one of the FE team members before shipping this.

3nvi reviewed (commented):
Hey @vorillaz thanks for that! The solution is great, but the spacing is a bit off. 

On the 2nd screenshot  the horizontal distance between the items is 5 times as big compared to the 1st screenshot.

Also if possible, can we apply the same solution to the following components:

- AlertDetailsInfo
- PolicyDetailsInfo

since they all implement  the "same logic"

vorillaz commented:
@3nvi You were right, updated the screenshots, fixed the column size according to the [new design grid](https://share.goabstract.com/58e615ef-9100-42ad-a85a-b0f21047c64e?mode=build&sha=a16f74b5775582ae7c69893be7e25659deef7678), and finally patched the other pages using the same specs.

3nvi reviewed (approved):
LGTM 🎉 

Haven't seen the other components, but SS  for alerts page looks correct  :)
